testkit/mockstore.go

```diff
@@ -59,6 +59,29 @@ func bootstrap(t testing.TB, store kv.Storage) (*domain.Domain, func()) {
 	return dom, clean
 }
 
+// CreateMockStoreAndDomainWithoutZeroSchemaLease return a new mock kv.Storage and *domain.Domain.
+func CreateMockStoreAndDomainWithoutZeroSchemaLease(t testing.TB, opts ...mockstore.MockTiKVStoreOption) (kv.Storage, *domain.Domain, func()) {
+	store, err := mockstore.NewMockStore(opts...)
+	require.NoError(t, err)
+	dom, clean := bootstrapWithoutZeroSchemaLease(t, store)
+	return store, dom, clean
+}
+
+func bootstrapWithoutZeroSchemaLease(t testing.TB, store kv.Storage) (*domain.Domain, func()) {
```

Comment:
I think you can pass a variable to set the lease instead and keep the default value to 0.